lib/virtfs/xfs/directory3_data_header.rb

```diff
@@ -1,7 +1,7 @@
-require 'fs/xfs/directory'
-require 'fs/xfs/superblock'
+require_relative 'directory'
+require_relative 'superblock'
```

Comment:
These lines should not be needed at all if the gem is loaded properly (i.e. they are loaded in virtfs/xfs.rb), so they can be remove entirely